CONTRIBUTING.md

```diff
@@ -86,7 +86,7 @@ also have important information on the PR workflow and the code style we use.
 #### Be nice to the git history
 
 Try to make simple PRs that handle one specific topic. Just like for reporting
-issues, it's better to open 3 different PRs that each address a different issue
+issues, it's better to open 3 different PRs that each addresses a different issue
```

Comment:
I'm not sure this should be singular. The subject is "3 different PRs", "each" is used as an adverb here. CC @cbscribe

Reply:
I'd change it completely to
`[...] it's better to open 3 different PRs, each addressing a different issue, than one [...]`